dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/message/NettyRequestMessage.java

```diff
@@ -308,33 +306,10 @@ public void destroy() {
     public boolean isBodyExpected() {
         boolean result = Boolean.FALSE;
```

Comment:
Can we just get rid of this now?

Reply:
Yeah nice catch, it was leftover so I'll remove this